web/templates/play/show.html.eex

```diff
@@ -1,2 +1,7 @@
 <%= render "_modal_keybinds.html" %>
 <div id="board-viewer" class="play-show"></div>
+
+<div class="view-btn-popup">
```

Comment:
This doesn't need to be here, as I said before i want to keep all links to game viewer one level above the individual resources.